Rakefile

```diff
@@ -177,3 +177,23 @@ task :validate do
     exit!
   end
 end
+
+desc 'Precompile assets'
+task :precompile do
+ require './lib/gollum/views/helpers.rb'
+ require './lib/gollum/assets.rb'
+ require 'sprockets'
+ require 'sprockets-helpers'
+ require 'sass'
+  env = Precious::Assets.sprockets
+  manifest = Sprockets::Manifest.new(env, './public/assets')
```

Comment:
It would be nice to add an option here to compile the assets to a custom location.

Reply:
👍